cucumber/api/features/support/rest_helpers.rb

```diff
@@ -99,6 +99,10 @@ def strip_rest_params_from_path path
   def set_result result
     @response_api_key = nil
     @status = result.code
+
+    # Authenticators helpers using http_status
+    @http_status = result.code
```

Comment:
Some helpers use `@status` some use `@http_status`. Same with `@result` and `@response_content`. I need to use helpers in order to avoid code dups and this is why I need to align with this.

Reply:
can't we align the usage between all helpers? so they all use only one?

Reply:
I I think that it is out of scope

Reply:
it really shouldn't be hard to do. just another small commit that will not require you to add a new variable.

Reply:
This is only 2 variables which are relevant to this PR. I think they might be more. This might affect all the integration tests and won't be in the scope of just one commit

Reply:
i really don't understand why we can't just replace these occurences from `@http_status` to `@status`.
![image](https://user-images.githubusercontent.com/29940926/91281819-31ccc900-e791-11ea-9794-a8209c52d0fb.png)

Reply:
In the common helpers it is being used without `@` 
![image](https://user-images.githubusercontent.com/31516429/91283102-d7346c80-e792-11ea-8ef2-6cd8f0e68fb8.png)
and so as `result`

Reply:
that still the same. it's a small find and replace in a separate commit.

Reply:
I agree that find and replace will do the work. But the main problem will be tests failing due to the change and if we change it. Also we should find other examples of dups like that  and it can be a good valid new. Again - out of scope of the PR.

Reply:
@orenbm If it's OK with you, lets fix it in a "tinny" PR, so we don't hold this feature before code freeze

Reply:
i'm fine with the use-case and against the idea in general. go for it.

Reply:
I already change to `http_status` in all api profile tests..... @uCatu